### ISSUE TITLE
Travis fails on unit test gen_external (not sure why)...

### ISSUE THREAD

**Issue by wdconinc** [JeffersonLab/remoll#110]
Seems like this may be caused by the `*new G4String` construct.

```
Writing output to remollin.root... written
### Run 1 starts.
### Run 1 start.
Initialized tree.

 *** Break *** segmentation violation
git repository info
-------------------------------------------------
commit 30aeee93a2439374eef9cabe07b4d7723ce7a68f
Author: Wouter Deconinck <[secure]@gmail.com>
Date:   Sat Jul 7 14:32:02 2018 -0400

    Create more atomic tar.gz files
    
    Now can download unit test analysis results without downloading the
    entire output root files. This will allow automatic test diffs with
    latest release.
## HEAD (no branch)

Geant4 version 10.3.1
  ROOT version 5.34/14
 cmake version 3.9.2

CMake suite maintained and supported by Kitware (kitware.com/cmake).

Generated at Sat Jul  7 18:41:58 UTC 2018
Source dir /home/travis/build/JeffersonLab/remoll/build
Build  dir /home/travis/build/JeffersonLab/remoll/build
-------------------------------------------------

Run at 2018-07-07 18:44:44 on travis-job-275c4331-0e8f-4a5c-8e7e-c88b7b791ef0
Run Path /home/travis/build/JeffersonLab/remoll
N generated = 1
Beam Energy = 0.000000 GeV
Generator   = external
Field maps:
Macro run:
-------------------------------------------------
Stored file /home/travis/build/JeffersonLab/remoll/macros/tests/unit/test_gen_external.mac (284 bytes)
/run/initialize
/remoll/evgen/set moller
/remoll/filename remollin.root
/run/beamOn 1000

/remoll/filename remollout.root

/remoll/evgen/set external
/run/beamOn 0
/remoll/evgen/external/file remollin.root
/remoll/evgen/external/detid 28
/remoll/filename remollout.root
/run/beamOn 1

-------------------------------------------------

Stored GDML Files:
	      geometry/mollerMother.gdml    2 kB
	    geometry/targetDaughter.gdml    2 kB
	  geometry/detectorDaughter.gdml    1 kB
	  geometry/trackingDaughter.gdml    1 kB
	geometry/pionDetectorSystem.gdml    4 kB
	geometry/pionDetectorVirtualDet.gdml    4 kB
	geometry/pionDetectorTSOpenSector.gdml    2 kB
	    geometry/pionDetectorTS.gdml    1 kB
	geometry/pionDetectorGEMOpenSector.gdml    2 kB
	   geometry/pionDetectorGEM.gdml    1 kB
	geometry/pionDetectorLuciteSector.gdml    5 kB
	geometry/pionDetectorLucite.gdml   29 kB
	geometry/pionDetectorCalorimeter.gdml    5 kB
	  geometry/upstreamDaughter.gdml   14 kB
	    geometry/hybridDaughter.gdml   47 kB
-------------------------------------------------

 Generating stack trace...
 0x00007f1616668e79 in TList::AddLast(TObject*) + 0x39 from /usr/lib/x86_64-linux-gnu/libCore.so.5.34
 0x00007f1616668572 in THashTable::Add(TObject*) + 0x62 from /usr/lib/x86_64-linux-gnu/libCore.so.5.34
 0x00007f161666c8c9 in TObjArray::Delete(char const*) + 0xe9 from /usr/lib/x86_64-linux-gnu/libCore.so.5.34
 0x00007f16166a2e7f in ROOT::TSchemaRule::ProcessList(TObjArray*, TString const&) + 0x2af from /usr/lib/x86_64-linux-gnu/libCore.so.5.34
 0x00007f16166a3846 in ROOT::TSchemaRule::SetFromRule(char const*) + 0x2b6 from /usr/lib/x86_64-linux-gnu/libCore.so.5.34
 0x00007f161668c20f in TClass::AddRule(char const*) + 0x3f from /usr/lib/x86_64-linux-gnu/libCore.so.5.34
 0x00007f161477cc4f in TFile::ReadStreamerInfo() + 0x33f from /usr/lib/x86_64-linux-gnu/libRIO.so.5.34
 0x00007f1614786e49 in TFile::Init(bool) + 0xc79 from /usr/lib/x86_64-linux-gnu/libRIO.so.5.34
 0x00007f16147879f3 in TFile::TFile(char const*, char const*, char const*, int) + 0x663 from /usr/lib/x86_64-linux-gnu/libRIO.so.5.34
 0x00007f161957de33 in remollGenExternal::SetGenExternalFile(G4String&) at /home/travis/build/JeffersonLab/remoll/src/remollGenExternal.cc:53 (discriminator 1) from /home/travis/build/JeffersonLab/remoll/build/libremoll.so
 0x00007f161957eca5 in G4AnyMethod::FuncRef1<void, remollGenExternal, G4String&>::operator()(void*, std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) at /usr/include/c++/4.8/sstream:565 from /home/travis/build/JeffersonLab/remoll/build/libremoll.so
 0x00007f16172cf9b3 in G4GenericMessenger::SetNewValue(G4UIcommand*, G4String) + 0x343 from /usr/lib/x86_64-linux-gnu/libG4intercoms.so.10.3
 0x00007f16172e2f84 in G4UIcommand::DoIt(G4String) + 0x1194 from /usr/lib/x86_64-linux-gnu/libG4intercoms.so.10.3
 0x00007f16172fdd29 in G4UImanager::ApplyCommand(char const*) + 0x7b9 from /usr/lib/x86_64-linux-gnu/libG4intercoms.so.10.3
 0x00007f161755c9db in G4WorkerRunManager::DoWork() + 0x2db from /usr/lib/x86_64-linux-gnu/libG4run.so.10.3
 0x00007f161756697c in G4MTRunManagerKernel::StartThread(void*) + 0x1ac from /usr/lib/x86_64-linux-gnu/libG4run.so.10.3
 0x00007f1610281184 in <unknown> from /lib/x86_64-linux-gnu/libpthread.so.0
 0x00007f1615cbfffd in clone + 0x6d from /lib/x86_64-linux-gnu/libc.so.6
```

**Comment by cameronc137** [comment]
Is this fixed by 83ab9cd81e5d72e54cf9723bc3453daf6c760efb?

**Comment by wdconinc** [comment]
Yes, https://travis-ci.org/JeffersonLab/remoll/builds/401808799.
